precompiles/avs/tx.go

```diff
@@ -33,3 +45,35 @@ func (p Precompile) AVSInfoRegisterOrDeregister(
 	}
 	return method.Outputs.Pack(true)
 }
+
+func (p Precompile) OperatorInfoRegister(
```

Comment:
The name of this function is confused, the operator is registered via the operator module, what you'd do is to register operator to avs, shouldn't this added in operator module?

Reply:
Also need to support operator deregister from avs

Reply:
fixed